components/aead/src/aes_gcm/mod.rs

```diff
@@ -60,6 +60,7 @@ impl<Ctx: Context> MpcAesGcm<Ctx> {
 }
 
 #[async_trait]
+#[allow(clippy::blocks_in_conditions)]
```

Comment:
this needs to be declared due to this upstream issue https://github.com/rust-lang/rust-clippy/issues/12281 — we shall file a ticket to remove this once the issue has been fixed